src/WordPress/Hooks.php

```diff
@@ -20,9 +20,19 @@ class Hooks
     const CLOUDFLARE_JSON = 'CLOUDFLARE_JSON';
     const WP_AJAX_ACTION = 'cloudflare_proxy';
 
+    // See https://developers.cloudflare.com/cache/about/default-cache-behavior/
+    const CLOUDFLARE_CACHABLE_EXTENSIONS = [
```

Comment:
A potential improvement, I wonder if this data is exposed via a URL, in which case we can get an extensions list at build time.

Reply:
They won't be changing often. It's safe enough to update them along with releases.